src/atom.ts

```diff
@@ -40,6 +90,8 @@ export function atom<Value, Update = unknown>(
     // are the same reference
     if (!equal(nextValue, getValue())) {
       subject(1, nextValue)
+      updateValue(myName, getValue())
```

Comment:
Hmm. It seems to work as intended so I'm obviously wrong, but - how come it shouldn't be `newValue` here instead of `getValue()`?

```suggestion
      updateValue(myName, newValue)
```